### ISSUE TITLE
Importing modules from same dir broken

### ISSUE THREAD

**Issue by neurotronix** [nteract/hydrogen#398]
After updating to 0.12.0 I am not able to import a python module I wrote (it sits in the same directory as the executed script). Firing up an ipython interpreter in the terminal and importing said module works like a charm though. I am on OS X 10.11.6, atom 1.9.6.
`apm --version` gives this output:
```
app  1.10.0
nam  2.13.3
node 0.10.40
python 3.5.2
git 2.7.4
```
I am using python from within an anaconda virtual environment, in case this might play a role.

**Comment by holgerabel** [comment]
+1 

apm  1.10.0
npm  2.13.3
node 0.10.40
python 2.7.12
git 2.9.2

**Comment by maurovargas** [comment]
same

apm  1.9.2
npm  2.13.3
node 0.10.40
python 2.7.11
git 2.7.4

**Comment by n-riesco** [comment]
Sorry about this regression. We've just released Hydrogen v0.12.1 to fix it.

**Comment by holgerabel** [comment]
Thank you!  I can confirm it works.
